packages/localization/src/config/translations.json

```diff
@@ -2704,6 +2704,10 @@
   "Mercury Mysteries": "Mercury Mysteries",
   "Venus Protectors": "Venus Protectors",
   "Uranus Unity Rain": "Uranus Unity Rain",
+  "MM Route": "MM Route",
```

Comment:
type check of i18n key now is failed, hard to review if missed key

Reply:
can you explain more?

Reply:
`t('xxx')` if xxx is not one key of the json

Reply:
It's not meant to be strict type safe by using (string || {}) for selection complete only

We have our own test cases for it

https://github.com/pancakeswap/pancake-frontend/blob/develop/apps/web/src/__tests__/translations.test.ts